pandas/core/resample.py

```diff
@@ -513,11 +513,20 @@ def _wrap_result(self, result):
         """
         Potentially wrap any results.
         """
+        obj = self.obj
+        if (
+            isinstance(result, ABCDataFrame)
+            and result.empty
+            and type(result.index) != PeriodIndex
```

Comment:
Can you use `not isinstance(result.index, PeriodIndex)`. Also, when the result is empty and the result.index is a PeriodIndex, what is the current behavior?

Reply:
The PeriodIndex is handled differently and the df is converted to a series. Check `test_resample_empty_dataframe` in `test_base.py`

Reply:
Great - thanks!